src/lm_buddy/integrations/wandb/artifact_utils.py

```diff
@@ -110,3 +110,38 @@ def build_table_artifact(
         table = wandb.Table(data=table_data, columns=columns)
         artifact.add(table, name=table_name)
     return artifact
+
+
+def build_file_artifact(
```

Comment:
I kept a structure very similar to `build_directory_artifact` and this might seem a bit redundant, so I am happy to think about alternatives.